src/main/java/unicash/ui/CommandBox.java

```diff
@@ -22,6 +32,12 @@ public class CommandBox extends UiPart<Region> {
     @FXML
     private TextField commandTextField;
 
+    // To keep a history of previous userInputs
+    private List<String> userInputHistory = new ArrayList<>();
+
+    // To keep track of which userInput the user is currently on
+    private int currentUserInputIndex = -1;
```

Comment:
Should we instead, use `null` instead of `-1` to indicate that the `currentUserInputIndex` has reached the limit? Just a very minor nit, to avoid the usage of magic values.

Reply:
Sure, I can do something about the magic value usage, but using null might be dangeroous given the chance of running into NullPointerExceptions. I have changed the `-1` to a class constant!